app/Controllers/feedController.php

```diff
@@ -794,7 +810,9 @@ public static function deleteFeed(int $feed_id): bool {
 
 			// Remove related queries
 			FreshRSS_Context::$user_conf->queries = remove_query_by_get(
-				'f_' . $feed_id, FreshRSS_Context::$user_conf->queries);
```

Comment:
formatting changes only; unfortunately these "escaped" my checks and were committed